.github/workflows/gradle.yml

```
@@ -21,6 +21,7 @@ permissions:
 env:
   LC_ALL: C.UTF-8
   CGO_ENABLED: 1
+  ENABLE_TRANSACTION_EXECUTOR_STEP: "false" #Enable when the full implementation of reusable services is completed
```

Shouldn't we just enable it and let it fail so we can see the progress being made? We can just not mark the check as required. Perhaps we only do it for v1 as well for now. We can set [continue-on-error](https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions#jobsjob_idstepscontinue-on-error) to not fail the job.

---

done